apollo-core/src/main/java/com/ctrip/framework/apollo/tracer/internals/NullMessageProducer.java

```diff
@@ -41,6 +41,16 @@ public void logEvent(String type, String name) {
   public void logEvent(String type, String name, String status, String nameValuePairs) {
   }
 
+  @Override
+  public void logMetricsForCount(String name) {
```

Comment:
i.e this NullMessageProducer don't change any code. even interface MessageProducer changed.